app/src/main/java/com/android/partagix/ui/components/ItemUi.kt

```diff
@@ -44,264 +48,469 @@ import androidx.compose.ui.text.style.TextOverflow
+            Text(text = user.rank, style = TextStyle(fontWeight = FontWeight(500)))
+            Spacer(modifier = Modifier.width(5.dp))
+            Text(
+              text = user.name,
+              style = TextStyle(
+                fontWeight = FontWeight(500),
+                textAlign = TextAlign.Left,
+              ),
+              maxLines = 1,
+              overflow = TextOverflow.Ellipsis,
+              modifier = Modifier.fillMaxWidth(0.95f)
+            )
+          }
+          Text(
+            text = if (loan.idItem == "") {
+              "not borrowed"
+            } else {
+              if (loan.startDate.before(Date())) {
+                "available until ${
+                  LocalDateTime.ofInstant(
+                    date.toInstant(), java.time.ZoneId.systemDefault()
+                  ).format(formatter)
+                }"
+              } else {
+                "borrowed until ${
+                  LocalDateTime.ofInstant(
+                    date.toInstant(), java.time.ZoneId.systemDefault()
+                  ).format(formatter)
+                }"
+              }
+            },
+            style = TextStyle(fontSize = 12.sp, letterSpacing = 0.25.sp),
+            modifier = Modifier.fillMaxWidth()
+          )
+        }
+        Column(
+          modifier = Modifier.requiredHeight(height = 64.dp), horizontalAlignment = Alignment.End
+        ) {
+          Text(
+            text = item.name,
+            textAlign = TextAlign.End,
+            style = TextStyle(textAlign = TextAlign.Right),
+            maxLines = 1,
+            overflow = TextOverflow.Ellipsis,
+            modifier = Modifier
+              .fillMaxWidth(.3f)
+              .padding(top = 10.dp)
+          )
+          Text(
+            text = "Quantity: " + item.quantity.toString(),
+            style = TextStyle(
+              fontSize = 11.sp,
+              textAlign = TextAlign.Right,
+            ),
+            textAlign = TextAlign.End,
+            lineHeight = 0.8.em,
+            modifier = Modifier
+              .fillMaxWidth(0.2f)
+              .padding(top = 5.dp)
+          )
+        }
+        Box(
+          modifier = Modifier
+            .fillMaxWidth(0.3f)
+            .fillMaxHeight()
+            .border(1.dp, MaterialTheme.colorScheme.outlineVariant)
+        ) {
+          AsyncImage(
+            model = item.imageId.absolutePath,
+            contentDescription = "fds",
+            contentScale = ContentScale.Inside,
+            modifier = Modifier
+              .fillMaxHeight()
+              .align(Alignment.Center)
+              .testTag("ItemUiImage")
+          )
+        }
+      }
+    }
+  }*/
+}
+
+@Composable
+fun ClickableText(contact: String) {
+  val context = LocalContext.current
+  Text(
+      text = contact,
+      fontSize = 11.sp,
+      lineHeight = 1.3.em,
+      fontWeight = FontWeight.Bold,
+      modifier =
+          Modifier.clickable {
+            val intent =
```

Comment:
Those Intents are really value to the app, having much more interactivity overall. Great job !